include/exadg/convection_diffusion/spatial_discretization/operator.cpp

```diff
@@ -323,6 +345,8 @@ Operator<dim, Number>::setup()
 
   if(param.use_cell_based_face_loops)
     Categorization::do_cell_based_loops(*grid->triangulation, mf_data->data);
+
+  mf->clear();
```

Comment:
probably not needed because default constructor is called a few lines above?

Reply:
yes, removed it!